src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -424,3 +449,29 @@ int2sub(1) -> pending;
 int2sub(2) -> unconfigured;
 int2sub(3) -> subscribed.
 
+item_to_record({NodeIdx, ItemId, CreatedLUser, CreatedLServer, CreatedAt,
+                ModifiedLUser, ModifiedLServer, ModifiedLResource, ModifiedAt,
+                PublisherIn, PayloadDB}) ->
+    PayloadXML = mongoose_rdbms:unescape_binary(CreatedLServer, PayloadDB),
```

Comment:
`global` instead of `CreatedLServer`?

Reply:
Right, this will do the trick. Thanks for pointing this out.